src/User/Factory/MailFactory.php

```diff
@@ -114,6 +114,28 @@ public static function makeReconfirmationMailerService(User $user, Token $token)
         return static::makeMailerService(MailEvent::TYPE_RECONFIRM, $from, $to, $subject, 'reconfirmation', $params);
     }
 
+    /**
+     * @param User  $user
+     * @param String $code
+     *
+     * @throws InvalidConfigException
+     * @return MailService
+     */
+    public static function makeTwoFactorCodeMailerService(User $user, String $code)
+    {
+        /** @var Module $module */
+        $module = Yii::$app->getModule('user');
+        $to = $user->email;
+        
+        $from = $module->mailParams['fromEmail'];
+        $subject = $module->mailParams['reconfirmationMailSubject'];
```

Comment:
Why does it use the `reconfirmationMailSubject` param? Shouldn't it use a dedicated parameter (e.g. "twoFactorMailSubject")? The email is sent with "Confirm email change on {application}" as the default subject.